_quarto.yml

```diff
@@ -50,8 +47,8 @@ format:
     classoption: [krantz2]
     fig-width: 6
     # fig-height: 9
-    format-resources:
-      - latex/krantz.cls
+    # format-resources:
```

Comment:
Is this intentional?

Reply:
Yes, though it wasn't the *correct* intention; reverted!